packages/tooltip/style/index.css

```diff
@@ -8,21 +8,17 @@
   border: var(--jp-border-width) solid var(--jp-border-color1);
   font-size: var(--jp-ui-font-size0);
   box-shadow: 0px 1px 6px rgba(0, 0, 0, 0.2);
-  max-width: 500px;
-  overflow: auto;
+  max-width: 650px;
```

Comment:
I sized it this way so 80 characters would fit with our default font.   Perhaps we should draw the hoverbox to the left if it goes offscreen?